exercises/transpose/example.tt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'transpose'
+
+# Test data version:
+# <%= sha1 %>
+class TransposeTest < Minitest::Test
+<% test_cases.each do |test_case| %>
+  def <%= test_case.test_name %>
+    <%= test_case.skipped %>
+    expect = <%= test_case.expect %>
+    actual = <%= test_case.work_load %>
+    assert_equal expect, actual
```

Comment:
I really like the 3 line format you're using here. 
I'd call it `expected` but `expect` is ok too.